### ISSUE TITLE
acc/com problem

### ISSUE THREAD

**Issue by AddASecond** [ToughStoneX/U-MVS#6]
Hi, have you validate the precision of this code/model that supposed to be align with the paper?
I try to reproduce with the setting of mvsnet backbone, using the pretrained.ckpt model + fusible + matlab evaluation code,
but the results is relatively low by a large margin:

|          |       acc |    com | Overall|
|paper|0.4695|0.4308|0.4501|
**|reproduce| 0.5728 |0.4978 |0.5353|**

**Comment by AddASecond** [comment]
Hi, I also try to reproduce U-MVS(casmvsnet backbone), using the pretrained.ckpt model + fusible + matlab evaluation code,
the result is also very different to the paper(with much lower acc but relatively higher com, overall is not as good as mentioned in the paper):
| | acc | com | Overall|
|paper|0.354|0.3535|0.3537|
**|reproduce| 0.4282 |0.3176 |0.3729|**

Can you explain about the difference?

**Comment by ToughStoneX** [comment]
Hi. 
I have tested it before uploading. May I ask you if you are using the same version of Pytorch as the posted one (Pytorch 1.2 or 1.3, same as the original [CasMVSNet](https://github.com/alibaba/cascade-stereo)) in Readme?
During the testing phase, I found out that if you are using Pytorch version over than 1.5 to load pretrained model trained by Pytorch with version under 1.5, performance drop will occur in the evaluation results.
The reason is that the saved model weights are loaded in a different way in Pytorch(over 1.5).
Note that similar issue is also discussed in CasMVSNet: https://github.com/alibaba/cascade-stereo#requirements.

**Comment by ToughStoneX** [comment]
I have just tested the code and generate the point clouds of DTU test set.
The score is:
| acc. | com. | overall |
| 0.4021 | 0.3046 | 0.3534 |
By simply modifying the hyperparameters in the provided script and trade off the performance between acc. and com. should be able to achieve the similar results as the paper.
Furthermore, here is the [download link](https://mogface.oss-cn-zhangjiakou.aliyuncs.com/xhb/share/umvs_iccv/u_mvs_camvsnet_results_dtu.zip) of the aforementioned generated point clouds and the temporary matlab files.
Maybe you can download it and check it out.
The .mat files also contain the acc. and comp. scores of each scene.
Hope this will help.

**Comment by AddASecond** [comment]
Hi, I've downgrade pytorch version to 1.4 (under 1.5, 1.3 won't even able to run the code), while the test result is still 0.02mm difference to your results, but similar to my results that using pytorch 1.9
|acc. |com. | overall |
| 0.42569615 | 0.32262542 | 0.37416078

may I makesure 2 questions:
1) did you use this repo to reproduce or your local code?
2) how to "simply" modifying the hyperparameters that reproduce the result in the paper?

I hope a more specific explain could be down，
because this really waste a lot of time due to the long pipeline，
and your answer can not answer all the doubts until now
